student-bios/Alexander-Segovia.md

```diff
@@ -0,0 +1,26 @@
+# Introducing myself
+
+<img src="https://i.redd.it/a2gfj1ookcm81.jpg" width="200" height="200" alt="my first pokemon">
```

Comment:
@Alexander-Segovia, the <img> tag is working fine and nice, but it is an html tag, so it could be easily changed with the markdown language version of the img tag, you can find it on the web by searching 'how to add img to markdown document'

Reply:
I was not able to change the image dimensions using Markdown, that is the only reason why I used HTML!